src/vs/platform/native/electron-main/nativeHostMainService.ts

```diff
@@ -56,6 +56,10 @@ export class NativeHostMainService extends Disposable implements INativeHostMain
 
 	declare readonly _serviceBrand: undefined;
 
+	private gpuInfoParentWindow: BrowserWindow | null = null;
```

Comment:
Why do we need to store these 2 windows as variables? If the intent is to focus when already opened, I would only store 1 variable, the identifier of the window and not the full window.

Reply:
Because `gpuInfoWindow` not `codeWindow` or `auxWindow`, I just want to make sure the `gpuInfoWindow` will be closed as long as the focused window gets closed, a single `gpuInfoWindowId` should be enough I will fix it later.